.travis.yml

```diff
@@ -5,8 +5,8 @@ env:
     # purposely specifying slowest builds first, to gain time overall
     - LMOD_VERSION=6.5.1
     - LMOD_VERSION=6.5.1 TEST_EASYBUILD_MODULE_SYNTAX=Tcl
-    - LMOD_VERSION=7.8.5
-    - LMOD_VERSION=7.8.5 TEST_EASYBUILD_MODULE_SYNTAX=Tcl
+    - LMOD_VERSION=7.8.22
```

Comment:
I chose `7.8.22` rather than `7.8.90` because I saw issues on my local machine with `7.8.90` (with my `Devel` module that is valid with every other release)

Reply:
7.8.22 makes sense, that's the actual latest 7.8.x based on https://github.com/TACC/Lmod/blob/master/README.old#L31, 7.8.90 is more like a 8.0 beta...